tests/assist.handler.spec.ts

```diff
@@ -83,6 +83,38 @@ describe("Assist handler", function () {
     expect(response.status).toBe(415);
   });
 
+  it("succeed on valid audio/ogg content-type", async () => {
+    // @ts-expect-error overriding read-only property
+    MockEvent.request.headers = new Map(
+      Object.entries({
+        "CF-Connecting-IP": "1.2.3.4",
+        "content-type": "audio/ogg",
+      })
+    );
+    const response = await routeRequest(MockSentry, MockEvent);
+    const result: Record<string, string> = await response.json();
+    expect(response.status).toBe(201);
+    expect(result.message).toStrictEqual("success");
+    expect(result.key.endsWith(".webm")).toBeTruthy();
```

Comment:
```suggestion
    expect(result.key.endsWith(".ogg")).toBeTruthy();
```
If accepted as-is this will fail, the key generation needs to change to set the correct file type.